pyuvsim/tests/test_telescope.py

```diff
@@ -209,24 +185,6 @@ def test_comparison(beam_objs):
     assert beamlist == beamlist2
 
 
-def test_no_overwrite(beam_objs):
-    if not hasattr(beam_objs[0], "_freq_interp_kind"):
-        # this can go away when we require pyuvdata version >= 2.4.2
-        pytest.skip()
```

Comment:
so we don't need this test at all? or just didn't need to make this check but should still keep the test?

Reply:
The functionality it's associated with has been removed. It was testing handling of `uvb_params`.